### PR TITLE
CHE-1819: fix NPE when unresolved project is deleted

### PR DESCRIPTION
**Pull request by svor**
### What does this PR do?

Fix NPE after deleting unresolved multi-module project. 
Show notification with the number of modules which were deleted.
### What issues does this PR fix or reference?

https://github.com/eclipse/che/issues/1819
### Previous Behavior

After deleting multi-module project the process of resolving dependencies is continued and threw many NPEs
### New Behavior

This content may also be included in the release notes.
After deleting multi-module project: project is deleted without exceptions and the process of resolving dependencies is ended 
### Tests written?

No

Please review @vparfonov

### CODE REVIEW

**Comment by vparfonov** [comment]
ok

File: ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/context/AppContextImpl.java [hunk 1]
```diff
@@ -414,6 +414,10 @@ public Project getRootProject() {
             }
         }
 
+        if (currentResources[0] == null) {
```
Comment (tolusha) [inline ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/context/AppContextImpl.java]: What does it actually mean?

Comment (svor) [inline ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/context/AppContextImpl.java]: It means that current resource (selected resource) is null. In this class 
`currentResources[0] == currentResource`
I've made some changes and now it's more clear, I hope.

Comment (tolusha) [inline ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/context/AppContextImpl.java]: It is better now. thx

**Pull request merged**